benches/macrobenches.rs

```diff
@@ -54,7 +54,7 @@ fn parse_document_from_str(doc: &str) -> XmlResult<()> {
                 }
             }
             Event::Text(e) => {
-                criterion::black_box(e.unescape()?);
+                criterion::black_box(e.decode()?);
```

Comment:
About `Reader` is returning merged events - does that mean that the `.decode()` goes away completely as the entities would be pre-expanded?  And would there be any way to, say, return the original raw unexpanded XML between two tags from that wrapper?

Reply:
Yes, I think, `decode` method will gone

> And would there be any way to, say, return the original raw unexpanded XML between two tags from that wrapper, or would that be impossible without dropping to the `RawReader` level?

I think, it could be possible to implement that, but only if something requested that. I think, that this is niche feature. The API could be a special method that need to call instead of `read_event()` to get an unparsed content.